src/components/DashboardMenu.tsx

```diff
@@ -29,43 +29,64 @@ const DashboardMenu: React.FC = () => {
       <menu className="courses">
         <ul className="courses_list">
           <li>
-            <TabButton onSelect={() => handleSelect("main-course")}>
+            <TabButton
+              onSelect={() => handleSelect("main-course")}
+              active={selectedCourse === "main-course"}
+            >
```

Comment:
create a variable for this "main-course" etc so that you dont have to write the string repeatedly

Reply:
(2)